config/database.yml

```diff
@@ -1,9 +1,9 @@
 default: &default
   adapter: postgresql
   encoding: utf8
-  host: localhost
-  username:
-  password:
+  host: <%= ENV.fetch('DB_HOST', 'localhost') %>
+  username: <%= ENV.fetch('POSTGRES_USER', 'twentyliters') %>
+  password: <%= ENV.fetch('POSTGRES_PASSWORD', 'password3214') %>
```

Comment:
Remove these defaults? I'd rather get an error message about not setting this variable than a weird message about this username and password.

Reply:
I could go either way on this - I just didn't want to break this for anyone who isn't using docker.

Reply:
Didn't realize we had dotenv-rails in the project, I'll go ahead and remove the defaults.

Reply:
I'm down with the fetch, but just not the 'twentyliters' and 'password3214'. I'm fine with forcing folks will need to set up a `.env` file.

Reply:
Removed the defaults and included an example .env file in this PR.